g3doc/quick_reference.md

```diff
@@ -786,6 +786,11 @@ All other ops in this section are only available if `HWY_TARGET != HWY_SCALAR`:
     `sum[i] + a[4*i+3]*b[4*i+3] + a[4*i+2]*b[4*i+2] + a[4*i+1]*b[4*i+1] +
     a[4*i+0]*b[4*i+0]`
 
+*   `VW`: `{u,i}{16,32,64}`: `V`: `{u,i}{8,16,32}`, `D`, 
+    `VW`: `Vec<RepartitionToWide<DFromV<V>>`:\
+    `VW WidenMulAccumulate(D, V, V, VW low, VW& high)`: widens `a` and `b`, \
```

Comment:
Let's add a and b to the parameters. How about something like this for the documentation?

widens `a` and `b`, multiplies them together, then adds them to the concatenated vectors `high:low`. Returns the lower half of the result, and sets `high` to the upper half.

Reply:
Thanks for suggestion! Added